### DISCUSSION TITLE
Table of contents for blog

### DISCUSSION THREAD

**Discussion by constantcopyandwork** [thuliteio/doks#761] [Ideas]
Would it be possible to add table of contents for the blog as well?

If I look for the code of it into the docs section, would it still work on the blog? Any tips on where to start looking?

Thanks!

**Comment by roheclassic** [comment]
In my opinion, replacing ```blog/single.html``` with ```docs/single.html``` is a very fast approach.

If you want to get closer to the original behavior, do the following.

- Change the ```<h1>{{ .Title }}</h1>``` around line 27 to the following:

```
      <div class="blog-header">
        <h1>{{ .Title }}</h1>
        {{ partial "main/blog-meta.html" . }}
      </div>
```

- Enclose lines ```<div class="blog-header">```  through ```{{.Content }}``` in ```<article>``` tags.

*If you don't use doc-menu, breadcrumb, docs-navigation, etc., please comment them out.
